src/Identity.ts

```diff
@@ -1,8 +1,76 @@
-// @flow
+// // this module should be used to represent an identity
```

Comment:
This is all commented out?

Reply:
This isn't used for anything so I've removed it